script/Ciel/Day 7/07_01_CIEL07_1C.txt

```diff
@@ -133,24 +141,24 @@ Putting that aside.
 -- 今日はともかくガツガツしたものが食べたい。
 Either way, today I feel like I want to eat something really filling.
 }
-[sha:e8980f16013ed6cbe5cbbede7bb7d7af686b018b]{
+[offset:15002]{
 -- Page 83, Offset 15002. Mods: @k, @e.
 -- ここは―――
-I―――
+I'll―――
 }
 [sha:7dc475597c54091ac1047dab43bf99fd31189d7b]{
 -- Page 84, Offset 42706. Choice.
 -- 食堂でカロリーの高い定食を頼もう。
- Go to the cafeteria and order something with a lot of calories.
+ Order something calorific at the cafeteria.
 }
 [sha:bc5e30b25d51412364b6ce7cb4c32d08388fef1d]{
 -- Page 84, Offset 42707. Choice.
 -- 購買でカレーパンを買い占めよう。
- Go to the school store and buy up a bunch of curry bread.
+ Buy up all the curry bread at the school store.
```

Comment:
'all' makes this come across too strongly

Reply:
I mean, he's a hungry lad. I think my main issue was with this formulation of 'buy up' - didn't sit right to me, feels like you have to buy up _on_ curry bread, or buy up _all_ the curry bread. I'll weaken it a bit.